### PR TITLE
Correction for the formal syntax not found issue - 02

### PR DESCRIPTION
**Pull request by ghost**
This is the 2th part for the issue #18780
In this PR I covered these instances:

- box-align
- box-direction
- box-flex
- box-flex-group
- box-lines
- box-ordinal-group
- box-orient
- box-pack
- font-smooth
- ime-mode
- scroll-snap-coordinate
- scroll-snap-destination
- scroll-snap-points-x
- scroll-snap-points-y
- scroll-snap-type-x
- scroll-snap-type-y
- zoom

#### Metadata

- [x] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/css/box-align/index.md [hunk 1]
```diff
@@ -66,7 +66,10 @@ If the alignment is set using the element's `align` attribute, then the style is
 
 ## Formal syntax
 
-{{csssyntax}}
+```plain
+box-align = 
+ start | center | end | baseline | stretch
```
Comment (wbamberg) [inline files/en-us/web/css/box-align/index.md:71]:
```suggestion
  start | center | end | baseline | stretch
```

File: files/en-us/web/css/box-direction/index.md [hunk 2]
```diff
@@ -51,7 +51,10 @@ If the direction is set using the element's `dir` attribute, then the style is i
 
 ## Formal definition
 
-{{cssinfo}}
+```plain
+box-direction =
+ normal | reverse | inherit
```
Comment (wbamberg) [inline files/en-us/web/css/box-direction/index.md:56]:
```suggestion
  normal | reverse | inherit
```

File: files/en-us/web/css/box-align/index.md (hunk 1 above)
Comment (wbamberg) [inline files/en-us/web/css/box-align/index.md:71]:
```suggestion
  start | center | end | baseline | stretch
```

File: files/en-us/web/css/box-flex/index.md [hunk 3]
```diff
@@ -64,7 +64,10 @@ A trick to make all content elements in a containing box the same size, is to gi
 
 ## Formal syntax
 
-{{csssyntax}}
+```plain
+box-flex =
+ <number>
```
Comment (wbamberg) [inline files/en-us/web/css/box-flex/index.md:69]:
```suggestion
  <number>
```

File: files/en-us/web/css/box-ordinal-group/index.md [hunk 4]
```diff
@@ -38,7 +38,10 @@ The `box-ordinal-group` property is specified as any positive {{CSSxRef("&lt;int
 
 ## Formal syntax
 
-{{CSSSyntax}}
+```plain
+box-ordinal-group =
+ <integer>
```
Comment (wbamberg) [inline files/en-us/web/css/box-ordinal-group/index.md:43]:
```suggestion
  <integer>
```

File: files/en-us/web/css/box-orient/index.md [hunk 5]
```diff
@@ -54,7 +54,10 @@ HTML DOM elements lay out their contents along the inline-axis by default. This
 
 ## Formal syntax
 
-{{csssyntax}}
+```plain
+box-orient =
+ horizontal | vertical | inline-axis | block-axis | inherit
```
Comment (wbamberg) [inline files/en-us/web/css/box-orient/index.md:59]:
```suggestion
  horizontal | vertical | inline-axis | block-axis | inherit
```

File: files/en-us/web/css/box-pack/index.md [hunk 6]
```diff
@@ -83,7 +83,10 @@ If the packing is set using the element's `pack` attribute, then the style is ig
 
 ## Formal syntax
 
-{{CSSSyntax}}
+```plain
+box-pack =
+ start | center | end | justify
```
Comment (wbamberg) [inline files/en-us/web/css/box-pack/index.md:88]:
```suggestion
  start | center | end | justify
```

File: files/en-us/web/css/font-smooth/index.md [hunk 7]
```diff
@@ -50,7 +50,10 @@ font-smooth: unset;
 
 ## Formal syntax
 
-{{csssyntax}}
+```plain
+font-smooth =
+ auto | never | always | <absolute-size> | <length>
```
Comment (wbamberg) [inline files/en-us/web/css/font-smooth/index.md:55]:
```suggestion
  auto | never | always | <absolute-size> | <length>
```

File: files/en-us/web/css/scroll-snap-destination/index.md [hunk 8]
```diff
@@ -37,7 +37,10 @@ scroll-snap-destination: unset;
 
 ## Formal syntax
 
-{{csssyntax}}
+```plain
+scroll-snap-destination =
+ <position>
```
Comment (wbamberg) [inline files/en-us/web/css/scroll-snap-destination/index.md:42]:
```suggestion
  <position>
```

File: files/en-us/web/css/scroll-snap-points-x/index.md [hunk 9]
```diff
@@ -42,7 +42,10 @@ scroll-snap-points-x: unset;
 
 ## Formal syntax
 
-{{csssyntax}}
+```plain
+scroll-snap-points-x =
+ none | repeat( <length-percentage> )
```
Comment (wbamberg) [inline files/en-us/web/css/scroll-snap-points-x/index.md:47]:
```suggestion
  none | repeat( <length-percentage> )
```

File: files/en-us/web/css/scroll-snap-points-y/index.md [hunk 10]
```diff
@@ -42,7 +42,10 @@ scroll-snap-points-y: unset;
 
 ## Formal syntax
 
-{{csssyntax}}
+```plain
+scroll-snap-points-y =
+ none | repeat( <length-percentage> )
```
Comment (wbamberg) [inline files/en-us/web/css/scroll-snap-points-y/index.md:47]:
```suggestion
  none | repeat( <length-percentage> )
```

File: files/en-us/web/css/scroll-snap-type-y/index.md [hunk 11]
```diff
@@ -46,7 +46,10 @@ scroll-snap-type-y: unset;
 
 ## Formal syntax
 
-{{CSSSyntax}}
+```plain
+scroll-snap-type-y =
+ none | mandatory | proximity
```
Comment (wbamberg) [inline files/en-us/web/css/scroll-snap-type-y/index.md:51]:
```suggestion
  none | mandatory | proximity
```

**Review by wbamberg — APPROVED** [review]
👍 thank you @najmiehsa !

**Comment by najmiehsa** [comment]
Thanks^^

**Pull request merged**